.goreleaser.yml

```diff
@@ -93,3 +93,47 @@ snapshot:
   name_template: "{{ .Tag }}-next"
 changelog:
   sort: asc
+dockers:
+- <<: &docker_opts
+    use_buildx: true
+    goos: linux
+  <<: &386
+    goarch: 386
+    image_templates:
+      - "ghcr.io/fastly/cli:{{ .Version }}-386"
+    build_flag_templates:
+      - "--platform=linux/386"
+  dockerfile: Dockerfile-assemblyscript
+- <<: *docker_opts
+  <<: &amd64
+    goarch: amd64
+    image_templates:
+      - "ghcr.io/fastly/cli:{{ .Version }}-amd64"
+    build_flag_templates:
+      - "--platform=linux/amd64"
+  dockerfile: Dockerfile-assemblyscript
+- <<: *docker_opts
+  <<: &arm64
+    goarch: arm64
+    image_templates:
+      - "ghcr.io/fastly/cli:{{ .Version }}-arm64"
+    build_flag_templates:
+      - "--platform=linux/arm/v7"
+  dockerfile: Dockerfile-assemblyscript
+- <<: *docker_opts
+  <<: *386
+  dockerfile: Dockerfile-rust
+- <<: *docker_opts
+  <<: *amd64
+  dockerfile: Dockerfile-rust
+- <<: *docker_opts
+  <<: *arm64
+  dockerfile: Dockerfile-rust
+  build_flag_templates:
+    - "--platform=linux/arm/v8"
+docker_manifests:
```

Comment:
Do we **_need_** to use `docker manifest`?

We have to use `dockers` any way to build the images, and it seems like it's possible to define multiple dockerfile files to be used when building these images, so I'm not sure what the value is in generating a manifest as well? 

Especially considering the trade-off is that 'manifests' is an experimental feature that, in some cases (depending on the version of Docker CLI that is running), needs to be explicitly enabled.

Hmm, thinking about it, I guess using this manifest feature would be useful from a 'workflow' perspective. 

**So for example**: I would imagine without a manifest the workflow, for people who want to use the dockerised version of the CLI, would be to visit the GitHub release page where (again I don't actually know how these docker images will be shown) the user will find the relevant image link to the `ghcr.io` endpoint. 

If they had the manifest, then an automated CI flow they could just use the Docker CLI to request a manifest for the CLI and then parse out the image version they're interested in.